src/app/pages/portal/portal.component.ts

```diff
@@ -493,9 +496,25 @@ export class PortalComponent implements OnInit, OnDestroy {
     this.workspaceEntitySortChange$.next(true);
   }
 
+  getNoQueryClickInWorkspace(result): boolean {
```

Comment:
If noQueryOnClickInTab is defined in workspace object, you could only check this condition on selectedWorkspace object.